test/pkg/environment/aws/default_ec2nodeclass.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: karpenter.k8s.aws/v1
+kind: EC2NodeClass
+metadata:
+  name: default
+spec:
+  amiFamily: AL2023
+  subnetSelectorTerms:
+    - tags: 
+        karpenter.sh/discovery: aengeda-karpenter-playground-us-west-2
```

Comment:
Does there need to be some form of templating for these tag values or does the value not matter?

Reply:
These have been updated